test/cypress/integration/agreements/agreement.cy.js

```diff
@@ -68,6 +68,7 @@ describe('Agreement detail page', () => {
     cy.get('[data-cy=agreement-detail]').find('dd').eq(2).contains('active')
     cy.get('[data-cy=agreement-detail]').find('dt').eq(3).contains('Description')
     cy.get('[data-cy=agreement-detail]').find('dd').eq(3).contains('Reason shown to paying user for taking agreement')
+    cy.get('[data-cy=agreement-detail]').find('dd').eq(3).contains("console.log('AGREEMENT')")
```

Comment:
Might be better if this contains the full `Reason shown to paying user for taking agreement <script>console.log('AGREEMENT')</script>` word so we know we’re sanitising the HTML tags rather than removing them.